### ISSUE TITLE
Add Voice Channel Member Counter

### ISSUE THREAD

**Issue by spdermn02** [spdermn02/TouchPortal_Discord_Plugin#67]
Add voice channel member counter 

will include the user as one of the members. No need to prevent that if > 1 that means another person joined you.

**Comment by gitagogaming** [comment]
I do not believe this is possible as we have to subscribe to the voice channel to be able to see how many users are inside of it.. and this would take ALOT of subscriptions to voice channels

**Comment by spdermn02** [comment]
But do we have a state for current voice channel count?

**Comment by gitagogaming** [comment]
> But do we have a state for current voice channel count?

Not currently no, there is just the "list" of total IDs right now that would be only way to tell total users
